plugins/woocommerce/client/admin/client/settings/settings.scss

```diff
@@ -0,0 +1,49 @@
+@include wordpress-admin-schemes();
```

Comment:
Once we have content areas, I'd like to fully work out the mobile scenarios in a follow up